caikit/runtime/service_factory.py

```diff
@@ -66,6 +67,20 @@
     }
 }
 
+RUNTIME_INFO_SERVICE_NAME = "RuntimeInfo"
+RUNTIME_INFO_SERVICE_SPEC = {
+    "service": {
+        "rpcs": [
+            {
+                "name": "GetServerInfo",
+                # TODO: this has no input param...but unsure how to codify that here since this json schema is required? str input_type? - was hitting validation errors in py_to_proto
+                "input_type": TrainingInfoRequest.get_proto_class().DESCRIPTOR.full_name,
```

Comment:
there is no input param for this endpoint so I wasn’t sure what to put for input_type . I tried deleting the line, having None , or setting an empty string but these all failed py_to_proto validation because I think [this](https://github.com/IBM/py-to-proto/blob/main/py_to_proto/json_to_service.py#L35) shows the expected schema which is a string value for the input_type?

Reply:
error message:
```
tests/runtime/conftest.py:157: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/usr/local/Cellar/python@3.11/3.11.3/Frameworks/Python.framework/Versions/3.11/lib/python3.11/contextlib.py:137: in __enter__
    return next(self.gen)
tests/runtime/conftest.py:147: in runtime_grpc_test_server
    with RuntimeGRPCServer(*args, **kwargs) as server:
caikit/runtime/grpc_server.py:125: in __init__
    ServicePackageFactory.get_service_package(
caikit/runtime/service_factory.py:133: in get_service_package
    grpc_service = json_to_service(
.....
  log.debug2("Validating service json")
        if not validate_jtd(json_service_def, SERVICE_JTD_SCHEMA, EXTENDED_TYPE_VALIDATORS):
>           raise ValueError("Invalid service json")
E           ValueError: Invalid service json
```

Reply:
Shoot, yeah there's an `Empty.proto` that would be designed for this but I don't think that's hooked up in `py_to_proto`. I'd try creating a `@dataobject` that has no attributes, and that way it's at least extendable in the future if we need to add anything. I could see taking a list of extra package names that you want to get the versions for.